portfolio/src/main/webapp/style.css

```diff
@@ -149,61 +149,71 @@ body {font-family: Arial, Helvetica, sans-serif;}
 
 /** Box format for the Places search bar */
 .search-reports {
-    margin: 10px 10px 0 0;
-    border-radius: 2px 0 0 2px;
-    box-sizing: border-box;
-    outline: none;
-    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
-    background-color: #fff;
-    font-family: Roboto;
-    display: block;
+  margin: 10px 10px 0 0;
+  border-radius: 2px 0 0 2px;
+  box-sizing: border-box;
+  outline: none;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+  background-color: #fff;
+  font-family: Roboto;
+  display: block;
 }
 
 #searchBox {
-    padding-top: 12px;
-    padding-bottom: 12px;
-    margin-right: 12px;
-    margin-left: 12px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  margin-right: 12px;
+  margin-left: 12px;
 }
 
 #search-title {
-    color: #fff;
-    background-color: #4d90fe;
-    font-size: 25px;
-    font-weight: 500;
-    padding: 6px 12px;
+  color: #fff;
+  background-color: #4d90fe;
+  font-size: 25px;
+  font-weight: 500;
+  padding: 6px 12px;
 }
 
 .search-route {
-    margin: 10px 10px 0 0;
-    border-radius: 2px 0 0 2px;
-    box-sizing: border-box;
-    outline: none;
-    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
-    background-color: #fff;
-    font-family: Roboto;
+  margin: 10px 10px 0 0;
+  border-radius: 2px 0 0 2px;
+  box-sizing: border-box;
+  outline: none;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+  background-color: #fff;
+  font-family: Roboto;
 }
 
 #originRoute {
-    padding-top: 12px;
-    padding-bottom: 12px;
-    margin-right: 12px;
-    margin-left: 12px;
+  padding-top: 12px;
+  padding-bottom: 12px;
+  margin-right: 12px;
+  margin-left: 12px;
 }
 
 #destinationRoute {
-    margin-left: 12px;
+  margin-left: 12px;
 }
 
 #search-route-title {
-    color: #fff;
-    background-color: #4d90fe;
-    font-size: 25px;
-    font-weight: 500;
-    padding: 6px 12px;
+  color: #fff;
+  background-color: #4d90fe;
+  font-size: 25px;
+  font-weight: 500;
+  padding: 6px 12px;
 }
 
 .search-filter {
-    display: inline-block;
-    padding: 5px 11px;
+  display: inline-block;
+  padding: 5px 11px;
+}
+
+#repeatedMarkerUI {
+  display: block;
```

Comment:
use `display: none` or `visibility: hidden` so the element starts out as hidden without having to call js. Then you can show it in your js when you need.